streamx-flink/streamx-flink-connector/streamx-flink-connector-doris/src/main/java/com/streamxhub/streamx/flink/connector/doris/bean/LoadStstusFailedException.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019 The StreamX Project
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.streamxhub.streamx.flink.connector.doris.bean;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class LoadStstusFailedException extends IOException {
```

Comment:
May be should LoadStatusFailedException?

Reply:
> May be should LoadStatusFailedException?

+1 😂

Reply:
i fix it